Tools/OSD/osd_screen.cs

```diff
@@ -604,12 +606,6 @@ private void rbtSortAlphabetic_CheckedChanged(object sender, EventArgs e)
 			}
 		}
 		
-        private int correct_NTSC(int y){
-            if (y >= osd.getCenter() && osd.is_ntsc()) 
-                y+=3;
-            return y;
-        }
-
```

Comment:
yes PRs are welcome - but not "I will bear everything and remake in my own way". Always some rules exists. Do you try to make PR to Ardupilot or Linux kernel?